<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindDialog</class>
 <widget class="QWidget" name="ConfigWindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wind</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,10">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="picLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GlopalParameter</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" columnminimumwidth="0,1,0,1,0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>AirDensity(Kg/m³)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="SelfDefLineEdit" name="aridensityLineEdit">
            <property name="text" stdset="0">
             <string>1.226</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_128">
            <property name="text">
             <string>AverageWindSpeed(m/s)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="SelfDefLineEdit" name="windspeedLineEdit">
            <property name="text" stdset="0">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_130">
            <property name="text">
             <string>WindAngle(°)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="SelfDefLineEdit" name="windangleLineEdit">
            <property name="text" stdset="0">
             <string>180</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="changespeekCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>CheckPulseSpeed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="windspeedGroupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>PulseSpeedSimulateParameter</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_132">
             <property name="text">
              <string>WindSpectrumInputType</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="windsComBox">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <item>
              <property name="text">
               <string>NPD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>API</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_133">
             <property name="text">
              <string>LowCutOffFrequency(Hz)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="SelfDefLineEdit" name="windminfrequencyLineEdit">
             <property name="text" stdset="0">
              <string>0.002</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_135">
             <property name="text">
              <string>HightCutOffFrequency(Hz)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="SelfDefLineEdit" name="windmaxfrequencyLineEdit">
             <property name="text" stdset="0">
              <string>0.1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="apitypeLabel">
             <property name="text">
              <string>ApiSpectral parameters(α)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="SelfDefLineEdit" name="apitypeLineEdit">
             <property name="text" stdset="0">
              <string>0.025</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_138">
             <property name="text">
              <string>WindSpeedHarmonicsNumber</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="windspeekcountSpinBox">
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_139">
             <property name="text">
              <string>DiscreteWindSpectrumNumber</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="windspnumberLineEdit">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>99</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_140">
             <property name="text">
              <string>RandomNumber</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="windrandomSpinBox">
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>99</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>WindComponent</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0,0,0,0">
        <item>
         <widget class="QTableWidget" name="windcomponentTableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>260</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>WindArea(m²)</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>WindFactor</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>VectorX</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>VectorY</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>VectorZ</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CentroidCoordinateComponent1</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CentroidCoordinateComponent2</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CentroidCoordinateComponent3</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>200.0000</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>1.0000</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>0.0000</string>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>0.0000</string>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string>1.0000</string>
           </property>
          </item>
          <item row="0" column="5">
           <property name="text">
            <string>0.0000</string>
           </property>
          </item>
          <item row="0" column="6">
           <property name="text">
            <string>0.0000</string>
           </property>
          </item>
          <item row="0" column="7">
           <property name="text">
            <string>20.0000</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_36">
          <property name="topMargin">
           <number>11</number>
          </property>
          <property name="rightMargin">
           <number>11</number>
          </property>
          <property name="bottomMargin">
           <number>11</number>
          </property>
          <item>
           <widget class="QLabel" name="label_189">
            <property name="text">
             <string>WindArea(m²)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="SelfDefLineEdit" name="windareaLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_269">
            <property name="text">
             <string>WindFactor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="SelfDefLineEdit" name="windfactorLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_26">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>WindCompoentThree</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_146">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="SelfDefLineEdit" name="windcoordinatesxLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_147">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="SelfDefLineEdit" name="windcoordinatesyLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_148">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="SelfDefLineEdit" name="windcoordinateszLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_24">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>LoadVectorDirection</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_143">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="SelfDefLineEdit" name="windvectorxLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_144">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="SelfDefLineEdit" name="windvectoryLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_145">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="SelfDefLineEdit" name="windvectorzLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_37">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="componentaddBtn">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="componenteditBtn">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="componentdelBtn">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="componenclearBtn">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SelfDefLineEdit</class>
   <extends>QWidget</extends>
   <header>selfdeflineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
